test/controllers/tickets_controller_test.rb

```diff
@@ -70,6 +71,18 @@ class TicketsControllerTest < ActionController::TestCase
 
   end
 
+  test 'should accept tickets in Base64 encoding' do
+    I18n.locale = :nl
+
+    assert_difference 'ActionMailer::Base.deliveries.size', User.agents.count do
+      assert_difference 'Ticket.count' do
+        post :create, message: @simple_base64_email, format: :json
```

Comment:
Right, I overlooked the regexp. Is the regexp really necessary for you? We also receive e-mails that are a few MB (because of attachments) and I don't think regexp matching such a big message is usefull.